java/src/main/native/src/ColumnViewJni.cpp

```diff
@@ -2452,7 +2452,14 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnView_getJSONObject(
     options.set_allow_single_quotes(allow_single_quotes);
     options.set_strip_quotes_from_single_strings(strip_quotes_from_single_strings);
     options.set_missing_fields_as_nulls(missing_fields_as_nulls);
-    return release_as_jlong(cudf::get_json_object(n_strings_col_view, *n_scalar_path, options));
+    std::unique_ptr<cudf::column> result_col_ptr;
+    try {
+      result_col_ptr = cudf::get_json_object(n_strings_col_view, *n_scalar_path, options);
+    } catch (std::invalid_argument const &err) {
+      auto const null_scalar = cudf::string_scalar(std::string(""), false);
+      result_col_ptr = cudf::make_column_from_scalar(null_scalar, n_strings_col_view.size());
+    }
+    return release_as_jlong(result_col_ptr);
```

Comment:
Variables should be assigned upon declaration. With multiple branches like this, use lambda:
```
auto result = [&] {
try {
      return cudf::get_json_object();
    } catch (std::invalid_argument const &err) {
      ..
      return cudf::make_column_from_scalar();
    } catch(...) {
       throw;
    }
}();
```

Notice that we have the additional `catch(...) { throw; }` to catch all other exceptions.